qemu_v8.mk

```diff
@@ -19,6 +19,15 @@ BR2_ROOTFS_POST_SCRIPT_ARGS = "$(QEMU_VIRTFS_AUTOMOUNT) $(QEMU_VIRTFS_MOUNTPOINT
 
 OPTEE_OS_PLATFORM = vexpress-qemu_armv8a
 
+################################################################################
+# If you change this, you MUST run `make clean` before rebuilding
```

Comment:
It might be worth mentioning which gits must be cleaned. Does the kernel need cleaning? Does buildroot need cleaning?

Reply:
Sure, I will add that. gits which need cleaning are mainly - optee_os, arm-tf, buildroot

Reply:
buildroot clean is not required. The xen scripts don't interfere with normal build. so, only arm-tf-clean and optee-os-clean needs to be done. I will update the comment.